src/components/campaignLandingPage/formatted-type.ts

```diff
@@ -21,4 +22,12 @@ export interface CampaignLandingPage extends PublishedEntity {
   whyThisMatters: string
   audience: { name: string }[]
   socialLinks: SocialLink[]
+  whatYouCanDo: {
+    header: string
+    intro: string
+    promos: {
+      image: MediaImage
+      link: ParagraphLinkTeaser
```

Comment:
This is a nitpick so take it or leave it, but I wonder if this should be flattened out in the query and not passed unformatted from Drupal. I don't think this is hurting anything but it just caught my eye that a Drupal type was used in the formatted type.